Drumpad.cpp

```diff
@@ -0,0 +1,10 @@
+#include "Drumpad.h"
+
+Drumpad::Drumpad(int num) 
+{
+    numPads = num;
+}
+
+Drumpad::~Drumpad() { 
+    std::cout << "Drumpad distroyed" << std::endl; 
+}
```

Comment:
Curly braces go on their own line.
Follow the course coding standard.
- [ ] revise